### DISCUSSION TITLE
AERORP, EYEPONT and VRP

### DISCUSSION THREAD

**Discussion by ingStudent** [JSBSim-Team/jsbsim#863] [Q&A]
Hi everyone,

Can someone please explain to me the meaning of the AERORP, EYEPOINT and VRP used in the metrics in order to model an aicraft on JSBSim and which frame they are related to, it's been a week I was searching for them but I didn't quite understand them. 

Thank U :)

**Comment by seanmcleod** [comment]
Quick question, will determine how much detail I go into describing the AERORP. Are you familiar with how the forces and moments are specified in the FDM?

**Reply by ingStudent** [reply]
Not really, I don't have that much background in aero but if you can give me details it would be great (I'm in an internship so the more details I have the best it is) Thank U.

**Comment by seanmcleod** [comment]
What are you planning on doing? Building a new JSBSim FDM from scratch, or just tweaking an existing FDM? Are you purely using JSBSim stand-alone or are you using it as part of FlightGear?

**Reply by ingStudent** [reply]
I'm building a new one from scratch for somme new disigns of VTOL and for now, I'm purely using JSBSim stand-alone.

**Reply by ingStudent** [reply]
So yeah, if you could explain them to me it would be great ;)

**Comment by seanmcleod** [comment]
Hmm, if you're building a FDM from scratch for some new VTOL design you're going to have to have some amount of flight dynamics background, whether that's a formal course, or self-study based on online resources/text books. 

Then make sure you read through the JSBSim reference manual as well.

We can't give you that full background in a couple of comments in this sort of discussion forum.

Some quick answers to your specific questions regarding AERORP, EYEPOINT and VRP.

From the JSBSim Reference Manual.

> Structural Frame
>This frame is a common manufacturer’s frame of reference and is used to 
define points on the aircraft such as the center of gravity, the locations of all the wheels, the pilot eye-point, point masses, thrusters, etc. Items in the JSBSim aircraft configuration file are located using this frame. In the structural frame the X-axis increases from the nose towards the tail, the Y-axis increases from the fuselage out towards the right (when looking forward from the cockpit), and of course the Z-axis then is positive upwards. Typically, the origin for this frame is near the front of the aircraft (at the tip of the nose, at the firewall, or in front of the nose some distance). 

Now JSBSim is a 6DOF (Degrees Of Freedom) simulation engine, which means there are 3 axes for forces and 3 axes of moments. Now when moments are defined they need to be defined relative to a MRP (Moment Reference Point). In JSBSim this point is defined by AERORP.

The EYEPOINT refers to the location of the pilot's eye in the structural frame. This isn't really used for flight dynamics purposes, except to be able to report accelerations that would be felt by the pilot given their offset from the cg of the aircraft.

The VRP (Visual Reference Point) also isn't used for flight dynamics purposes, but it's used in coordination between software like FlightGear when using JSBSim so that when FlightGear or similiar software needs to render it's graphics view and placement of the 3D model of the aircraft in the world that they're in sync when JSBSim sends updates of the current lat, lon etc.

**Reply by ingStudent** [reply]
Thank U for your answer. I was searching on some good hints to know what to search about in detail and you give them to me. About the FDM and other stuff I think I could have a good help from the team I'm working with, the one probleme is that non of them used JSBSim and that's my part.

**Reply by ingStudent** [reply]
For the manual I m also currently on it on order to understand more.

**Comment by seanmcleod** [comment]
> About the FDM and other stuff I think I could have a good help from the team I'm working with

Yes, if they have general flight dynamics experience and are supplying the aerodynamic data etc. but they just need help with understanding/matching terminology, e.g. MRP (Moment Reference Point) vs AERORP and some help on taking standard aerodynamic data and formatting it for JSBSim etc. then you could provide them with answers based on questions you post here.
